api/src/main/java/liquibase/ext/sqlgenerator/core/ModifyColumnGenerator.java

```diff
@@ -192,12 +192,10 @@ String getPostDataTypeString(Database database) {
 	String getDefaultClause(ColumnConfig column, Database database) {
 		String clause = "";
 		String defaultValue = column.getDefaultValue();
-		if (defaultValue != null) {
-			if (database instanceof MySQLDatabase) {
+		if (defaultValue != null && database instanceof MySQLDatabase){
 				clause += " DEFAULT " + DataTypeFactory.getInstance().fromObject(defaultValue, database)
 					.objectToSql(defaultValue, database);
```

Comment:
Check the formatting here, there seems to be one unnecessary space at the beginning.

Reply:
Even here no space except if am not seeing well , that code is in the same line as before

Reply:
Line 195 - 2 tabs, line 196 - 4 tabs, right? Isn't it better if there were 3 instead of 4? Unless GitHub is distorting and showing something different than what you see in the IDE.

Reply:
![image](https://user-images.githubusercontent.com/9057882/91221882-a7e41800-e71e-11ea-9c3f-63aa7395b607.png)